### PR TITLE
Delete member cascade

### LINKED ISSUE #104: Deletion of `Person` does not remove references of the deleted person
Currently, the deletion of persons (i.e. `ModelManager#deletePerson()` method) does not remove associations to this person automatically.

This cascading behaviour should be enforced at the `Model` level, so we do not run into a situation where `ModelManager#deletePerson()` leaves the `Person` association hanging for `Task#assignees` and `Team#members`

### PR DESCRIPTION
**Pull request by Devanshshah1309**
### Changes
1. Change datetime format to be more user-friendly.

Fixes #104

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -29,8 +29,7 @@ public class AddressBook implements ReadOnlyAddressBook {
      *
      * Note that non-static init blocks are not recommended to use. There are other ways to avoid duplication
      *   among constructors.
-     */
-    {
+     */ {
```
Comment (Devanshshah1309) [inline src/main/java/seedu/address/model/AddressBook.java:32]:
```suggestion
     */
     {
```

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -30,6 +30,7 @@ public class AddressBook implements ReadOnlyAddressBook {
      * Note that non-static init blocks are not recommended to use. There are other ways to avoid duplication
      *   among constructors.
      */
+
```
Comment (Devanshshah1309) [inline src/main/java/seedu/address/model/AddressBook.java:33]:
```suggestion

```

Comment (Devanshshah1309) [inline src/main/java/seedu/address/model/AddressBook.java:33]: Ignore this while reviewing. For some reason, my IntelliJ keeps auto-indenting this comment block. As long as we pass checkstyle tests, it should be fine ig

**Review by changjy44 — APPROVED** [review]
LGTM Features and changes working as expected.

**Pull request merged**